tests/test_app.py

```diff
@@ -105,12 +105,16 @@ def test_app_configure_with_platforms(mocked_engine):
 
 @pytest.mark.usefixtures('mocked_engine')
 @mock.patch('bottery.app.asyncio')
-def test_app_run(mocked_asyncio):
+def test_app_run_n_stop(mocked_asyncio):
     """Should create tasks and run forever"""
 
     app = App()
     app.run()
+    app.stop()
+
     mocked_event_loop = mocked_asyncio.get_event_loop.return_value
 
     mocked_asyncio.get_event_loop.assert_called_with()
     mocked_event_loop.run_forever.assert_called_with()
+
+    mocked_event_loop.close.assert_called_with()
```

Comment:
I believe this test is incomplete (or should be another one). We are not testing that the close method is being called after a `KeyboardInterrupt` exception.